Http/Controllers/Backend/Advanced/BatchesController.php

```diff
@@ -15,11 +15,10 @@ public function __construct()
     //----------------------------------------------------------
     public function getAssets(Request $request): JsonResponse
     {
-        if (!Auth::user()->hasPermission('has-access-of-advanced-section')) {
```

Comment:
@we-vikram-wri231 writing `has-access-of-advanced-section` text multiple time, no point